internal/cmd/stack/flags.go

```diff
@@ -32,6 +32,12 @@ var flagNoFindRepositoryRoot = &cli.BoolFlag{
 	Value: false,
 }
 
+var flagProjectRootOnly = &cli.BoolFlag{
+	Name:  "project-root-only",
```

Comment:
Totally open to flag naming - I have zero opinions here - but this made sense to me, so if there is a broader strategy - or a better name I'm all ears.